.github/workflows/lint.yml

```diff
@@ -2,7 +2,7 @@ name: "Lint"
 
 on:
   push:
-    branches: [main,github-actions-test]
+    branches: [main,github-actions-test,ghaction-speedup-iss215]
```

Comment:
This should not come into the main branch, please remove from this pull request

Reply:
I have made the change. Sorry for this being left over!